src/peft/tuners/lora/config.py

```diff
@@ -69,6 +69,31 @@ class LoftQConfig:
     loftq_iter: int = field(default=1, metadata={"help": "Alternating iterations for LoftQ"})
 
 
+@dataclass
+class EvaConfig:
+    """
+    This is the sub-configuration class to store the configuration for a data-drive initialization via EVA.
```

Comment:
Let's add a link to the paper here as well. Also, what's always nice to have for users is to give some indication when/how to change the parameters. E.g. "increase rho if ...".